test/client/fixtures/automation/automations-test.js

```diff
@@ -383,7 +383,9 @@ $(document).ready(function () {
 
         window.async.series({
             'Click on textarea': function (callback) {
-                runClickAutomation($textarea[0], { caretPos: 5 }, callback);
+                runClickAutomation($textarea[0], { caretPos: 5 }, function () {
+                    callback();
```

Comment:
`clickAutomation.run` promise returns `eventArgs` now.
If we write
```js
runClickAutomation($textarea[0], { caretPos: 5 }, callback);
```
it pass `eventArgs` to a callback as a first argument. But `window.async` [expects](http://caolan.github.io/async/docs.html#series) that the first argument of the `callback` is an error. So we should call `callback()` without arguments